### PR TITLE
Change in twitter link reporting security issues

### PR DESCRIPTION
**Pull request by NayanshiSingh**
In the Report the security issues section the twitter link was not updated :
Screenshot 2024-08-15 224248

The updated link:

Screenshot 2024-08-15 224314

### CODE REVIEW

**Comment by gasman** [comment]
Thanks @NayanshiSingh! There are a couple of problems here, though:

* This pull request contains some unrelated changes - your previous ones from #12205, and an edit to third_party_tutorials.md. This probably happened because you made your earlier changes on the 'main' branch, and these were copied into the new branch when you started it. When you start work on a new PR, you should make sure that the 'main' branch is in sync with the original Wagtail repository with no additional commits - if you're using the command-line `git` program, you can do this with
```
git checkout main
git reset origin/main
git pull origin main
```

When you open a pull request, Github will show you a summary of the changes - please check these carefully, and confirm that they're the changes you intended to make.

* Please use a more informative commit message. See https://github.com/wagtail/wagtail/commits/main/ for some examples - almost everything in that list is resolving an issue of some sort, so a commit message saying just "resolved the issue" is not providing any useful information.

File: docs/advanced_topics/third_party_tutorials.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ the latest Wagtail versions.
 -   [Wagtail-Multilingual: a simple project to demonstrate how multilingual is implemented](https://github.com/cristovao-alves/Wagtail-Multilingual) (31 January 2017)
 -   [Wagtail: 2 Steps for Adding Pages Outside of the CMS](https://www.caktusgroup.com/blog/2016/02/15/wagtail-2-steps-adding-pages-outside-cms/) (15 February 2016)
 -   [Deploying Wagtail to Heroku](https://wagtail.org/blog/deploying-wagtail-heroku/) (15 July 2015)
--   [Adding a Twitter Widget for Wagtail's new StreamField](https://jossingram.wordpress.com/2015/04/02/adding-a-twitter-widget-for-wagtails-new-streamfield/) (2 April 2015)
+-   [Adding a  Widget for Wagtail's new StreamField](https://jossingram.wordpress.com/2015/04/02/adding-a-twitter-widget-for-wagtails-new-streamfield/) (2 April 2015)
```
Comment (lb-) [inline docs/advanced_topics/third_party_tutorials.md:154]:
```suggestion
-   [Adding a Widget for Wagtail's new StreamField](https://jossingram.wordpress.com/2015/04/02/adding-a-twitter-widget-for-wagtails-new-streamfield/) (2 April 2015)
```

Just a small whitespace clean up needed.

Comment (gasman) [reply docs/advanced_topics/third_party_tutorials.md:154]: I don't think we should change this - even if the brand name is outdated, it's still the one used in the article title.

Comment (lb-) [reply docs/advanced_topics/third_party_tutorials.md:154]: Agreed. Good call.

**Comment by gasman** [comment]
Now handled in 7051363f9f8a22688363ad8fcd9117267b6a83a8.

**Pull request closed without merging**